### DISCUSSION TITLE
errror: C:\Users\ehsan\Desktop\bot_uni\node_modules\Telegraf\lib\composer.js: TypeError: handler is not a function

### DISCUSSION THREAD

**Discussion by EhsanKey** [telegraf/telegraf#1794] [Q&A]
Hello friends
I met with a bug or maybe Error
I found the problem from here but I don't know how to handle it

Error text:
 C:\Users\ehsan\Desktop\bot_uni\node_modules\Telegraf\lib\composer.js:474
                await handler(context, async (ctx = context) => {
                      ^

TypeError: handler is not a function 

My code:

const _adedProfessor = require("./adedProfessor/");

module.exports = async function (bot) {
  await bot.use(_adedProfessor(bot));

  return bot;
};

_______________________________________________

module.exports = async function adedProfessor(bot) {
  console.log("TEST");
  return bot;
};

**Comment by maslakovSaveliy** [comment]
I have some problem too

**Reply by MKRhere** [reply]
OP's code looks weird because they're trying to mount bot on bot. Post your code and full error message.

**Reply by maslakovSaveliy** [reply]
Oh yeah, I already noticed. I have already solved my problem. I just imported the composer incorrectly :)
But thanks anyway <3
